src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,63 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 706px;
+  padding: 10px;
+  gap: 1px;
+
+  @each $day, $margin in $start-day {
+    &--start-day-#{$day} {
+      & :first-child {
+        margin-left: $margin;
+      }
+    }
+  }
+
+  @for $lastDay from 28 through 31 {
+    &--month-length-#{$lastDay} {
+      & :nth-child(#{$lastDay + 1}) {
+        visibility: hidden;
+      }
+      & :nth-child(#{$lastDay + 2}) {
+        visibility: hidden;
+      }
+      & :nth-child(#{$lastDay + 3}) {
+        visibility: hidden;
+      }
+    }
+  }
+
+  &__day {
+    display: flex;
+    background-color: #eee;
+    width: $box-size;
```

Comment:
The variable `$box-size` is used here, but it is not defined within this file. Ensure that it is defined elsewhere or imported correctly to avoid compilation errors.